api-reference/10 UI Components/GridBase/1 Configuration/syncLookupFilterValues.md

```diff
@@ -5,7 +5,25 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to show only relevant values in the [header filter]({basewidgetpath}/Configuration/headerFilter/) and [filter row]({basewidgetpath}/Configuration/filterRow/).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The following table shows how the component behaves when you assign different values to this property. The 'State' column is filtered by the 'Alabama' value. If you set the **syncLookupFilterValues** to `false`, the 'City' column's header filter and filter row display all the cities instead of showing cities that belong to Alabama only. 
```

Comment:
```suggestion
The following table shows how the component behaves when you assign different values to this property. The 'State' column is filtered by the 'Alabama' value. If you set the **syncLookupFilterValues** to `false`, the 'City' column's header filter and filter row display all cities instead of showing cities within Alabama only.
```